pyiron_potentialfit/spgfit/util.py

```diff
@@ -167,3 +168,29 @@ def symlink_project(pr: Project):
     if target_dir[-1] == "/":
         target_dir = target_dir[:-1]
     pr.symlink(os.path.join(SYMLINK_TARGET, os.path.dirname(target_dir)))
+
+
+# Generic config objects for all tools
+
+@dataclass
+class ServerConfig:
```

Comment:
Can this somehow be linked to the data classes used for the output? https://github.com/pyiron-dev/pyiron-hdf5-format/blob/main/pyiron_io/dataclasses/generic.py#L84

Reply:
Not so easily, because I'd like the dataclass defined here to include only those flags that are reasonably configured by a user/from CLI, though we could split the class you linked into two and and use the base class here.